cmd/kube-proxy/app/server_test.go

```diff
@@ -526,7 +526,7 @@ udpIdleTimeout: 250ms`)
 		}()
 
 		if tc.append {
-			file.WriteString("append fake content")
+			file.WriteString("udpIdleTimeout: 250ms")
```

Comment:
Do you think we should add more test cases for the new events, current one is only covering the WRITE event appending a line to the config.
Some use cases can be:
"Creating a new config file with the same content and check it doesn't restart"
"Creating a new config file with different and valid content and check it doesn't restart"
"Creating a new config file with different and invalid content and check it doesn't restart"
"Delete the config file and check it doesn't restart"
also liggit's one https://github.com/kubernetes/kubernetes/pull/86869#discussion_r364228619
> I wouldn't expect us to restart when the config file is removed